sdk/src/hash.rs

```diff
@@ -96,6 +97,44 @@ pub fn extend_and_hash(id: &Hash, val: &[u8]) -> Hash {
     hash(&hash_data)
 }
 
+// TODO: Merge this logic with pubkey::solana{,_name}_id
```

Comment:
No new TODOs please, file an issue instead

Reply:
And link in comment or no refs in code at all?  I saw the other tags added to nits, but what's policy going forward?

Reply:
Link to a github issue in the code if you want, but then that's also not necessary because we typically go from github issue -> code

Reply:
So ideally we have a github issue that has a perma-link to the "TODO" part of the code and that's good enough to track without having a code base littered with TODO/TBD/WIP/FIXME droppings

Reply:
Gotcha.  I guess I like the in-line notice, but understand that there's less visibility and motivation to actually address them there.

Reply:
Inline link to the github issue wfm too.   The goal is just to ensure we surface all into Github Issues and remove the additional subterranean layer of "TODO issues" that nobody beyond the original author really knows about